### PR TITLE
openblas: 0.3.13 -> 0.3.15

### PR DESCRIPTION
**Pull request by asfishman**
###### Motivation for this change
Openblas version 0.3.13 and 0.3.14 fails to properly identify the architecture on some ARMV8 systems. This is documented in the openblas change log for version 0.3.15 as a bugfix for ARM64 archs: https://github.com/xianyi/OpenBLAS/blob/v0.3.15/Changelog.txt#L40

The fix changelog note reads:

> fixed a segmentation fault in DYNAMIC_ARCH builds (reappeared in 0.3.14) 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Ubuntu 18.04 multiuser nixpkgs
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126114` at 88765346 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>47 packages marked as broken and skipped:</summary>
  <ul>
    <li>blastem</li>
    <li>cassandra_2_1</li>
    <li>cassandra_2_2</li>
    <li>curaByDagoma</li>
    <li>cura_stable</li>
    <li>displaycal</li>
    <li>gnuradio3_7</li>
    <li>gnuradio3_7Packages.ais</li>
    <li>gnuradio3_7Packages.gnuradio</li>
    <li>gnuradio3_7Packages.gsm</li>
    <li>gnuradio3_7Packages.limesdr</li>
    <li>gnuradio3_7Packages.nacl</li>
    <li>gnuradio3_7Packages.osmosdr</li>
    <li>gnuradio3_7Packages.rds</li>
    <li>googleearth-pro</li>
    <li>grass</li>
    <li>hadoop</li>
    <li>hadoop_2_7</li>
    <li>hadoop_2_8</li>
    <li>hadoop_2_9</li>
    <li>hplipWithPlugin_3_16_11</li>
    <li>hplipWithPlugin_3_18_5</li>
    <li>hplip_3_16_11</li>
    <li>hplip_3_18_5</li>
    <li>impressive</li>
    <li>invoice2data</li>
    <li>julia</li>
    <li>julia-lts</li>
    <li>julia_10</li>
    <li>linuxPackages_hardkernel_4_14.virtualbox</li>
    <li>linuxPackages_hardkernel_latest.virtualbox</li>
    <li>loxodo</li>
    <li>metamorphose2</li>
    <li>ocropus</li>
    <li>odpdown</li>
    <li>opentsdb</li>
    <li>pdfread</li>
    <li>pig</li>
    <li>plover.stable</li>
    <li>rainbowstream</li>
    <li>scribus</li>
    <li>scyther</li>
    <li>spark</li>
    <li>termplay</li>
    <li>torchat</li>
    <li>winpdb</li>
    <li>yaxg</li>
  </ul>
</details>
<details>
  <summary>137 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/66103f6a497fe6c6e75af1a33bd050ef">arpack</a></li>
    <li><a href="https://gist.github.com/7616a49880b904485fa1554f1448694c">auditBlasHook</a></li>
    <li><a href="https://gist.github.com/0efdd58753bbfb537a95888c53ce2bc0">black</a> (python38Packages.black)</li>
    <li><a href="https://gist.github.com/b3ea9f8cd3d9c1f5f39fefe8ce5d5f15">blas</a></li>
    <li><a href="https://gist.github.com/82c8d3505771c2a6af9796caf69b7564">csdp</a></li>
    <li><a href="https://gist.github.com/cd7e729ecfe12f237108ae9e943ce7bb">lapack</a></li>
    <li><a href="https://gist.github.com/41714d1175801f5aca76a581b5850149">openblas</a></li>
    <li><a href="https://gist.github.com/05d94b0a94a6570f12abc735ff564e98">openblasCompat</a></li>
    <li><a href="https://gist.github.com/d8229f66ced94badbb3171b9a781b74f">python38Packages.Rtree</a></li>
    <li><a href="https://gist.github.com/3839625e5b86128e37f233237932dd42">python38Packages.aiohttp</a></li>
    <li><a href="https://gist.github.com/d1610e3044496628f696b6e571e77243">python38Packages.aiohttp-cors</a></li>
    <li><a href="https://gist.github.com/c49608c302a01d31f8bf90f07d5dc697">python38Packages.aioresponses</a></li>
    <li><a href="https://gist.github.com/e3ef4d1952fb04fe564639780ea9fdee">python38Packages.arrayqueues</a></li>
    <li><a href="https://gist.github.com/c911dc984865b0ddd01a51060d58c339">python38Packages.backports_functools_lru_cache</a></li>
    <li><a href="https://gist.github.com/ebd91b13a9e195af69f8d594fc626876">python38Packages.bokeh</a></li>
    <li><a href="https://gist.github.com/ff14aa7205a86afd590dc684cdc457d5">python38Packages.boto</a></li>
    <li><a href="https://gist.github.com/93ad8b923472d7accc5c03d5f4e72978">python38Packages.bottleneck</a></li>
    <li><a href="https://gist.github.com/c8a351cdd9fab34c7d7da97977fa876f">python38Packages.brotlipy</a></li>
    <li><a href="https://gist.github.com/03f77a0516ab6b5cbbff8c75a1c9b4e5">python38Packages.colorcet</a></li>
    <li><a href="https://gist.github.com/c91ce318b4b4a138459c1fb3122416f7">python38Packages.colorspacious</a></li>
    <li><a href="https://gist.github.com/90e4e1ed8d144849500c0c959c2df05d">python38Packages.construct</a></li>
    <li><a href="https://gist.github.com/c02d1b71c715fe86ee0fe571b0d74c7d">python38Packages.descartes</a></li>
    <li><a href="https://gist.github.com/86bc82a49dd3637717b63e4b71b76d3f">python38Packages.emcee</a></li>
    <li><a href="https://gist.github.com/813b63413f02e3aac41f043c3e30267a">python38Packages.fastprogress</a></li>
    <li><a href="https://gist.github.com/14d2f9fc48cff8f276ded3264e1a73b2">python38Packages.folium</a></li>
    <li><a href="https://gist.github.com/69c718f7dd077ee5323f534ec088dbfa">python38Packages.fsspec</a></li>
    <li><a href="https://gist.github.com/774770d05c80d533312a5fa13c5589ce">python38Packages.google-api-core</a></li>
    <li><a href="https://gist.github.com/0602ba5e505f95428a81dbf5e47c65e1">python38Packages.google-auth</a></li>
    <li><a href="https://gist.github.com/a9d4205fbf7d9f6f7e8acfa5752ff661">python38Packages.google-auth-oauthlib</a></li>
    <li><a href="https://gist.github.com/52b335d4b75ad7f55e4ee93f34185ed3">python38Packages.google-cloud-core</a></li>
    <li><a href="https://gist.github.com/3d921de37ac10718682c5c4d5714e852">python38Packages.google-cloud-iam</a></li>
    <li><a href="https://gist.github.com/ecd89430daaa75aa0a27bc7efe9d37a8">python38Packages.google-cloud-kms</a></li>
    <li><a href="https://gist.github.com/0238699d894123675abdd63e8b55f795">python38Packages.google-cloud-storage</a></li>
    <li><a href="https://gist.github.com/16765f4a50ef93f33db7451442e6e385">python38Packages.google-cloud-testutils</a></li>
    <li><a href="https://gist.github.com/b394cace7a02ea9ef4f5f2e33c15baa8">python38Packages.google-resumable-media</a></li>
    <li><a href="https://gist.github.com/988b29968528d6461cf0b5a4f8340eb9">python38Packages.httpbin</a></li>
    <li><a href="https://gist.github.com/fae964df5a9c30f6e3b5b45487bb81d2">python38Packages.httpcore</a></li>
    <li><a href="https://gist.github.com/912c5491da0cffc261907641ee39431b">python38Packages.httplib2</a></li>
    <li><a href="https://gist.github.com/539d7e3a13ad77759a4eff64216fd1ae">python38Packages.httpretty</a></li>
    <li><a href="https://gist.github.com/ec0794f1a6a17e5c88c70addd4e77d66">python38Packages.httpx</a></li>
    <li><a href="https://gist.github.com/f42b46e4246eb0350cc9efe12dddc733">python38Packages.imagecodecs-lite</a></li>
    <li><a href="https://gist.github.com/9550754353d4f6d14943576c1b838947">python38Packages.imageio</a></li>
    <li><a href="https://gist.github.com/0e080efcd1a1a15d3643a5018416c28f">python38Packages.jaraco_collections</a></li>
    <li><a href="https://gist.github.com/f1aa9896f17acc661606272871c1bc14">python38Packages.jaraco_functools</a></li>
    <li><a href="https://gist.github.com/d9f99d2908c64b6c35dc045de6f10150">python38Packages.jaraco_text</a></li>
    <li><a href="https://gist.github.com/8b199e15fa749496ecd6714d6640d871">python38Packages.matplotlib</a></li>
    <li><a href="https://gist.github.com/997ad5f46b86b5f8d19e320d9dc65784">python38Packages.nose-randomly</a></li>
    <li><a href="https://gist.github.com/3888baeb5aef30e1794be5208e12d720">python38Packages.numba</a></li>
    <li><a href="https://gist.github.com/7aa22595d8e1c021144713db023578e7">python38Packages.numexpr</a></li>
    <li><a href="https://gist.github.com/f452c8688f7a3413d13651f19aca6cb3">python38Packages.numpy</a></li>
    <li><a href="https://gist.github.com/52bfabb9050332951c9f4b0757306215">python38Packages.oauth2client</a></li>
    <li><a href="https://gist.github.com/de4cae0884f702695c943ee5b8f9b4c9">python38Packages.pillow</a></li>
    <li><a href="https://gist.github.com/e77f24cff9bac8ee22f3def010e33e76">python38Packages.portend</a></li>
    <li><a href="https://gist.github.com/cfaaacd592750da0d9c9247d2084e933">python38Packages.pproxy</a></li>
    <li><a href="https://gist.github.com/674aaf5c55524c6dcb794db31b47dcf7">python38Packages.pyerfa</a></li>
    <li><a href="https://gist.github.com/4585dbb1e61edf4c42cf23a49960baeb">python38Packages.pyopengl</a></li>
    <li><a href="https://gist.github.com/afeab4f0b8293137a5f9286b54014f25">python38Packages.pyrr</a></li>
    <li><a href="https://gist.github.com/3ea40feff456ede14314168aaf7108ad">python38Packages.soundfile</a> (python38Packages.pysoundfile)</li>
    <li><a href="https://gist.github.com/67955c8d8c1ea34c000e8e391c6fe3a8">python38Packages.pytest-arraydiff</a></li>
    <li><a href="https://gist.github.com/8cd9b39fd2a3b28ad18afaefd0062ced">python38Packages.pytest-black</a></li>
    <li><a href="https://gist.github.com/479462428552c28c87020ca80508ccfc">python38Packages.pytest-doctestplus</a></li>
    <li><a href="https://gist.github.com/291f9c098b62e4426a870841579fe480">python38Packages.pytest-filter-subpackage</a></li>
    <li><a href="https://gist.github.com/639bb3cd9a929873f41a7e1a40782a62">python38Packages.pytest-httpbin</a></li>
    <li><a href="https://gist.github.com/9700106ed9759ed9e8e40dc7863a400e">python38Packages.pytest-mpl</a></li>
    <li><a href="https://gist.github.com/6b607c953680d98c1b3a87c2d9bb0d1b">python38Packages.pytest-randomly</a></li>
    <li><a href="https://gist.github.com/eaf3874b7c9ca1cdeb45987b8675ce8a">python38Packages.pytest-vcr</a></li>
    <li><a href="https://gist.github.com/ec760d21ab0f5ed2a2c98c6f4e4266b4">python38Packages.pytools</a></li>
    <li><a href="https://gist.github.com/b0eb6495d9957f4338dc020913994cf8">python38Packages.qimage2ndarray</a></li>
    <li><a href="https://gist.github.com/cf8fc5cabc8cce8e601b3b920063917f">python38Packages.reportlab</a></li>
    <li><a href="https://gist.github.com/06a262afe0be182fbfe8546b02bb3313">python38Packages.shapely</a></li>
    <li><a href="https://gist.github.com/353d0df6012b1aaf341d6d2eda2e5fed">python38Packages.smart-open</a></li>
    <li><a href="https://gist.github.com/d2e685b17032c6892fc43e2371002a99">python38Packages.sounddevice</a></li>
    <li><a href="https://gist.github.com/1257becd0f626e9abbb7d3dd292f2041">python38Packages.spglib</a></li>
    <li><a href="https://gist.github.com/37c83f7454e010c06dd612a800b44839">python38Packages.srsly</a></li>
    <li><a href="https://gist.github.com/8e18e67e79d93d12ed5f0287b20dea19">python38Packages.tables</a></li>
    <li><a href="https://gist.github.com/637566bdf47ff85bbf5a4e7538f37b58">python38Packages.tempora</a></li>
    <li><a href="https://gist.github.com/dd6a4d0e669c25da0cb175d5a005446b">python38Packages.tifffile</a></li>
    <li><a href="https://gist.github.com/294faeb92ded33268811f9031872bb22">python38Packages.traits</a></li>
    <li><a href="https://gist.github.com/57d2101f159478f2a1a371d63bbbfe9d">python38Packages.uncertainties</a></li>
    <li><a href="https://gist.github.com/fc8021ca6d6877f0cef336122c15b88f">python38Packages.uvicorn</a></li>
    <li><a href="https://gist.github.com/22f91c73b509799e5ad29dd2643418b8">python38Packages.uvloop</a></li>
    <li><a href="https://gist.github.com/ef0a5512a9cc2036520cfc08e8e8864c">python38Packages.vcrpy</a></li>
    <li><a href="https://gist.github.com/ec1324dc7c9b76ea2de21a0fe10a4bf2">python39Packages.Rtree</a></li>
    <li><a href="https://gist.github.com/1ad5a633ad75fbbc67abd3b515c3b852">python39Packages.aiohttp</a></li>
    <li><a href="https://gist.github.com/367d922d036bf7c93de965445b365d20">python39Packages.aiohttp-cors</a></li>
    <li><a href="https://gist.github.com/db8a7321538ef74590f9807d01f1fbc7">python39Packages.aioresponses</a></li>
    <li><a href="https://gist.github.com/a90fa56a196fc2a164db99c7709f7e60">python39Packages.backports_functools_lru_cache</a></li>
    <li><a href="https://gist.github.com/95d40d4f58c9aad158ef0286921601ec">python39Packages.black</a></li>
    <li><a href="https://gist.github.com/9b6f2ec4b6588ed7fd6bbafced0fb7af">python39Packages.bokeh</a></li>
    <li><a href="https://gist.github.com/a468ba811f445a453fc04d5b8c4fb84f">python39Packages.bottleneck</a></li>
    <li><a href="https://gist.github.com/4edfda442e61cd032aa57029f589eb39">python39Packages.brotlipy</a></li>
    <li><a href="https://gist.github.com/54c65ac111b73d5524746efd6de27fce">python39Packages.colorcet</a></li>
    <li><a href="https://gist.github.com/38938aee6ff2ae00e5946c2463c102a1">python39Packages.construct</a></li>
    <li><a href="https://gist.github.com/ffe29b0b6815c7be7bf32790e788f3f7">python39Packages.descartes</a></li>
    <li><a href="https://gist.github.com/7f90d801eb26b9ec083a9722b72642c6">python39Packages.emcee</a></li>
    <li><a href="https://gist.github.com/c9a71a25f2cad11476bbfd668a735129">python39Packages.folium</a></li>
    <li><a href="https://gist.github.com/af720304b45ed75b0823b41786207659">python39Packages.fsspec</a></li>
    <li><a href="https://gist.github.com/6e2432243906600852bfe0a6fca6199f">python39Packages.google-auth</a></li>
    <li><a href="https://gist.github.com/87f23b219e78e2fed33043160765e919">python39Packages.google-auth-oauthlib</a></li>
    <li><a href="https://gist.github.com/36e67fdcebab2a65b9d76c1b4a6ff6e2">python39Packages.httpbin</a></li>
    <li><a href="https://gist.github.com/23acdfaec658454f478f908cfe14bec0">python39Packages.httplib2</a></li>
    <li><a href="https://gist.github.com/8e48d0aa202261ab973f657c4ff56592">python39Packages.httpretty</a></li>
    <li><a href="https://gist.github.com/ed7c2233472ce1b46c63b2d022bf1065">python39Packages.imagecodecs-lite</a></li>
    <li><a href="https://gist.github.com/8da05da4ef1b18edf7b09212eeadfae5">python39Packages.jaraco_collections</a></li>
    <li><a href="https://gist.github.com/ad9201be499661414e2c1a2f1df6ac44">python39Packages.jaraco_functools</a></li>
    <li><a href="https://gist.github.com/f0d119217afe772615efcaca9a63c13d">python39Packages.jaraco_text</a></li>
    <li><a href="https://gist.github.com/65411a09d29052ddf47b896caba8e97e">python39Packages.matplotlib</a></li>
    <li><a href="https://gist.github.com/10bc5fd6c0b66c1064d42f3783da5214">python39Packages.nose-randomly</a></li>
    <li><a href="https://gist.github.com/ee352491e78ec491a66c9b805db89dcf">python39Packages.numexpr</a></li>
    <li><a href="https://gist.github.com/a03f6de8548c982af091ba3266b1e648">python39Packages.numpy</a></li>
    <li><a href="https://gist.github.com/2b2a9fcd223208ecd6a5db18f41e1b53">python39Packages.oauth2client</a></li>
    <li><a href="https://gist.github.com/d1291dbfcebf64c5f083de47105d7bae">python39Packages.pillow</a></li>
    <li><a href="https://gist.github.com/9a82028470d9885c10b7505b9cd05999">python39Packages.portend</a></li>
    <li><a href="https://gist.github.com/43afc67bf082e00314a4802ba507ff8a">python39Packages.pyerfa</a></li>
    <li><a href="https://gist.github.com/98c572a21b4904041b26b7ed542b00a9">python39Packages.pytest-arraydiff</a></li>
    <li><a href="https://gist.github.com/c6591e97ea9de3ade60ec9ee141f4995">python39Packages.pytest-black</a></li>
    <li><a href="https://gist.github.com/36c1f10c46f63ad80f441225ec87919e">python39Packages.pytest-doctestplus</a></li>
    <li><a href="https://gist.github.com/979000e93e71bd9d384855d3a8740040">python39Packages.pytest-filter-subpackage</a></li>
    <li><a href="https://gist.github.com/4a723bd0a62dc2627ef3318077be093d">python39Packages.pytest-httpbin</a></li>
    <li><a href="https://gist.github.com/e0380db23bff8ad19f1e179995dc4477">python39Packages.pytest-mpl</a></li>
    <li><a href="https://gist.github.com/5acb4a3823a801471a03c2f7c1331434">python39Packages.pytest-randomly</a></li>
    <li><a href="https://gist.github.com/9636668423496290f8be4799badc7246">python39Packages.pytest-vcr</a></li>
    <li><a href="https://gist.github.com/396f8e97fb5afa08d84c3629b088280a">python39Packages.pytools</a></li>
    <li><a href="https://gist.github.com/d2c9ffb2848d55aa97afd4382866abe8">python39Packages.reportlab</a></li>
    <li><a href="https://gist.github.com/4d007664c7f157defad45c5d41d6241a">python39Packages.shapely</a></li>
    <li><a href="https://gist.github.com/41fa7edc2773a5d19d1523a5c091ec49">python39Packages.spglib</a></li>
    <li><a href="https://gist.github.com/8739685dfe38ef172ef1f9c12e883c3e">python39Packages.tables</a></li>
    <li><a href="https://gist.github.com/aea588f573be8c6822cbdfea00f6759d">python39Packages.tempora</a></li>
    <li><a href="https://gist.github.com/0eacbb1f44d6d1604ef03a585006ab1f">python39Packages.tifffile</a></li>
    <li><a href="https://gist.github.com/150a7ddcff4e43e8c9c6d23ebad4427a">python39Packages.traits</a></li>
    <li><a href="https://gist.github.com/0638ea41a260cbc79ea658c54f3bd1f4">python39Packages.uncertainties</a></li>
    <li><a href="https://gist.github.com/10c70ed3c97fab6f9cc2a852aef887fd">python39Packages.uvicorn</a></li>
    <li><a href="https://gist.github.com/670a1740b2316e28f766ff81c9507079">python39Packages.uvloop</a></li>
    <li><a href="https://gist.github.com/53824c8e8c7affa3d9a7197bf9a250d9">python39Packages.vcrpy</a></li>
    <li><a href="https://gist.github.com/f803982751eede6d5aff61abd79985bb">qrupdate</a></li>
    <li><a href="https://gist.github.com/c100e2b1a29196317015bc0bb4058d40">suitesparse</a> (suitesparse_5_3)</li>
    <li><a href="https://gist.github.com/0b2ca61fae8639aa34cb14c516ba35a9">sundials</a></li>
  </ul>
</details>
<details>
  <summary>3142 packages skipped due to time constraints:</summary>
  <ul>
    <li>DisnixWebService</li>
    <li>MACS2</li>
    <li>MIDIVisualizer</li>
    <li>R</li>
    <li>abcl</li>
    <li>actdiag (python38Packages.actdiag)</li>
    <li>adapta-gtk-theme</li>
    <li>adoptopenjdk-icedtea-web</li>
    <li>aiodnsbrute</li>
    <li>alloy (alloy5)</li>
    <li>...</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

File: pkgs/development/libraries/science/math/openblas/default.nix [hunk 1]
```diff
@@ -116,15 +116,15 @@ let
...
 
   src = fetchFromGitHub {
     owner = "xianyi";
     repo = "OpenBLAS";
     rev = "v${version}";
-    sha256 = "14jxh0v3jfbw4mfjx4mcz4dd51lyq7pqvh9k8dg94539ypzjr2lj";
+    sha256 = "1qjr02cqncv20abdp1yzr55n7smhx6h9chqvb0xbp18byynvj87w";
   };
 
   # apply https://github.com/xianyi/OpenBLAS/pull/3060 to fix a crash on arm
```
Comment (rmcgibbo) [inline pkgs/development/libraries/science/math/openblas/default.nix:130]: it looks like this issue was already patched, so we need to remove this patch

Comment (rmcgibbo) [reply pkgs/development/libraries/science/math/openblas/default.nix:130]: also, can you change the commit message format? https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes is the standard.

**Review by rmcgibbo — CHANGES_REQUESTED** [review]
Thanks for doing this! Just two things:

**Comment by risicle** [comment]
This will have to go to `staging` because of the massive rebuild count.

**Comment by rmcgibbo** [comment]
I think the ideal target is `staging` (https://github.com/NixOS/nixpkgs/tree/staging(

File: pkgs/development/libraries/science/math/openblas/default.nix [hunk 2]
```diff
@@ -116,29 +116,17 @@ let
...
     repo = "OpenBLAS";
     rev = "v${version}";
-    sha256 = "14jxh0v3jfbw4mfjx4mcz4dd51lyq7pqvh9k8dg94539ypzjr2lj";
+    sha256 = "1qjr02cqncv20abdp1yzr55n7smhx6h9chqvb0xbp18byynvj87w";
   };
 
-  # apply https://github.com/xianyi/OpenBLAS/pull/3060 to fix a crash on arm
-  # remove this when updating to 0.3.14 or newer
-  patches = [
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/science/math/openblas/default.nix:133]: Please remove the unused fetchpatch from inputs.

**Comment by ajs124** [comment]
This is already in staging with #122035, assuming it wasn't reverted.

**Comment by SuperSandro2000** [comment]
No, it wasn't reverted. https://github.com/NixOS/nixpkgs/blob/staging/pkgs/development/libraries/science/math/openblas/default.nix#L125

**Pull request closed without merging**